pages/api/id/[form]/owners.ts

```diff
@@ -55,12 +58,16 @@ export async function activateOrDeactivateFormOwners(
 ): Promise<void> {
   //Extracting req body
   const requestBody = req.body ? JSON.parse(req.body) : undefined;
-  //Payload validation
-  if (!requestBody?.email || !requestBody?.active || typeof requestBody.active !== "boolean") {
+  //Payload validation fix: true case scenario
+  if (
+    !requestBody?.email ||
+    requestBody.active === undefined ||
```

Comment:
you could probably just remove this statement all together. if active is undefined typeof will yield undefined